README.md

```diff
@@ -1,2 +1,42 @@
 # setup-artemis-borgs
```

Comment:
Maybe update the title to match the style used in the `setup-artemis` repo?

Reply:
done.